keanu-python/keanu/vertex.py

```diff
@@ -34,9 +119,9 @@ def __parse_args(args):
     @staticmethod
     def __parse_arg(arg):
         if isinstance(arg, np.ndarray):
```

Comment:
This logic of checking the type for `numpy` arrays and checking for `number` and casting that to a rank 2 `numpy` array feels identical to the logic in the constructor of `Const`. Can we pull it out into a method?